src/3d/spread3d_wrapper.cu

```diff
@@ -452,7 +463,19 @@ int cuspread3d_blockgather_prop(int nf1, int nf2, int nf3, int M,
 	int o_bin_size_x = d_plan->opts.gpu_obinsizex;
 	int o_bin_size_y = d_plan->opts.gpu_obinsizey;
 	int o_bin_size_z = d_plan->opts.gpu_obinsizez;
+	
 	int numobins[3];
+	if (nf1 % o_bin_size_x != 0 ||
```

Comment:
Is this check (and the one below) still necessary given the fact that we make sure to have the correct `nf*`?

Reply:
I leave it there because there is function that only does the step of spreading (`cufinufft_spread3d`) and there, `nf*` is an input from users.

Reply:
Ok sounds good.